src/hooks/useCds/index.js

```diff
@@ -144,6 +145,14 @@ const applyCds = async function(patientData, setOutput, setIsLoadingCdsData, isT
       decisionAids = { errors };
     }
 
+    // replace with actual logging data when ready from CQL 
+    // output otherResources as a temporary stand-in
+    logMsg({
```

Comment:
Let's put `console.timeEnd('Apply CDS');` (currently towards the bottom of this file) before the logger call, since this is intended to measure the time the dashboard takes just to run the CDS, and its output may be skewed if it also includes the logging time.

Reply:
Could we add a console time statement for the logger, so we can see how much total time it is taking?

Reply:
> Let's put `console.timeEnd('Apply CDS');` (currently towards the bottom of this file) before the logger call, since this is intended to measure the time the dashboard takes just to run the CDS, and its output may be skewed if it also includes the logging time.

Done https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/82/commits/9409a68acceee0a074dbe0cb76bfe87c094bd37d

Reply:
Thanks for moving up the `console.timeEnd('Apply CDS');` statement. I saw you also added a console timer in the `logMsg` function itself. Nice!